op-challenger/config/config.go

```diff
@@ -254,6 +263,14 @@ func (c Config) Check() error {
 		if c.Asterisc.Server == "" {
 			return ErrMissingAsteriscServer
 		}
+
+		if _, err := os.Stat(c.Asterisc.VmBin); err != nil {
+			return fmt.Errorf("%w: %v", ErrMissingAsteriscBin, err)
```

Comment:
There are some linting failures here, looks like this should be:
```suggestion
			return fmt.Errorf("%w: %w", ErrMissingAsteriscBin, err)
```